src/components/sdk/MeetingControls/docs/VideoInputBackgroundBlurControl.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import VideoInputBackgroundBlurControl from '../VideoInputBackgroundBlurControl';
+
+<Meta title="SDK Components/MeetingControls/VideoInputBackgroundBlurControl" />
+
+# VideoInputBackgroundBlurControl
+
+The `VideoInputBackgroundBlurControl` component renders a `ControlBarButton` with pop over menu options to select through multiple video input sources.
+These video input sources are provided by the `useVideoInputs` hook.
+The `VideoInputBackgroundBlurControl` component also has a checkbox that will enable or disable Background Blur on the chosen video input source.
+If you want to apply Background Blur to your video input device stream, try this `VideoInputBackgroundBlurControl`. If not, `VideoInputControl` is a better choice.
+
+This component should be used within `BackgroundBlurProvider` and `MeetingProvider`.
+
+When you click the button, the local video input toggles between on/off state.
+
+You can use it in the [ControlBar component](/docs/ui-components-controlbar--control-bar) to build the meeting controls bar.
+
+## Importing
+
+```javascript
+import { VideoInputBackgroundBlurControl } from 'amazon-chime-sdk-component-library-react';
+```
+
+## Usage
+
+```jsx
+import React from 'react';
+import {
+  MeetingProvider,
+  VideoInputBackgroundBlurControl
+} from 'amazon-chime-sdk-component-library-react';
+
+const App = () => {
+  return (
+    <MeetingProvider>
+      <ControlBar
+        layout="undocked-horizontal"
+        showLabels
+      >
+        <AudioInputVFControl />
```

Comment:
We can change this to `AudioInputControl` or wrap `MeetingProvider` within `VoiceFocusProvider` if using `AudioInputVFControl`.